cranelift/codegen/meta/src/isa/x86/legalize.rs

```diff
@@ -341,17 +369,20 @@ pub(crate) fn define(shared: &mut SharedDefinitions, x86_instructions: &Instruct
     .isa("x86")
     .chain_with(shared.transform_groups.by_name("narrow_flags").id);
 
-    // SIMD
+    // Set up variables and immediates.
     let uimm8_zero = Literal::constant(&imm.uimm8, 0x00);
     let uimm8_one = Literal::constant(&imm.uimm8, 0x01);
     let u128_zeroes = constant(vec![0x00; 16]);
     let u128_ones = constant(vec![0xff; 16]);
+    let a = var("a");
     let b = var("b");
     let c = var("c");
     let d = var("d");
     let e = var("e");
+    let x = var("x");
+    let y = var("y");
 
-    // SIMD vector size: eventually multiple vector sizes may be supported but for now only SSE-sized vectors are available
+    // Limit the SIMD vector size: eventually multiple vector sizes may be supported but for now only SSE-sized vectors are available.
```

Comment:
Please keep comments wrapped at 100 columns or so (which is the rustfmt column limit, except it doesn't automatically split up comments).